R/build-home.R

```diff
@@ -120,6 +121,40 @@
 #'     href: http://website.com
 #' ```
 #'
+#' You can provide a ready-made sidebar HTML:
+#'
+#' ```
+#' home:
+#'   sidebar:
+#'     html: path-to-sidebar.html
+#' ```
+#'
+#' You can change the order of sidebar elements:
+#' links, license (with an "s"), community, citation, authors, dev (badges)
+#' and add custom ones.
+#' The example below creates a sidebar whose only elements will be the
+#' authors section, a custom section, and a Dev Status section if there are
+#' badges.
+#' Note that the html fiekd should be HTML formatted so if you add a link, you need a
+#' `<a>` tag.``
```

Comment:
I think you can now drop this line since it's obvious that a field named html will contain HTML 😄